src/plugins/arkeo/StakingCards.tsx

```diff
@@ -42,25 +34,7 @@ export const StakingCards: React.FC<StakingCardsProps> = ({ ids }) => {
     selectFirstAccountIdByChainId(state, osmosisChainId),
   )
 
-  const { cosmosSdkStakingOpportunities: cosmosStakingOpportunities } = useCosmosSdkStakingBalances(
-    {
-      assetId: cosmosAssetId,
-      accountId: cosmosAccountId,
-    },
-  )
-
-  const { cosmosSdkStakingOpportunities: osmosisStakingOpportunities } =
-    useCosmosSdkStakingBalances({
-      assetId: osmosisAssetId,
-      accountId: osmosisAccountId,
-    })
-
-  const cosmos = useMemo(
-    () => cosmosStakingOpportunities.concat(osmosisStakingOpportunities),
-    [cosmosStakingOpportunities, osmosisStakingOpportunities],
-  )
-
-  const allOpportunities = [...stakingOpportunities, ...useTransformCosmosStaking(cosmos)]
+  const allOpportunities = stakingOpportunities
```

Comment:
random leftover assignment?

Reply:
Mostly diff purposes but might as well ditch it bff9a04650633695321a3b406cf20fefc542a918